cmd/mistry/main.go

```diff
@@ -139,6 +128,31 @@ EXAMPLES:
 					Usage:       "Output the build result in JSON format to STDOUT (implies verbose: false)",
 					Destination: &jsonResult,
 				},
+
+				// transport flags
+				cli.BoolFlag{
+					Name:        "async",
+					Usage:       "If set, don't wait for the build to finish and exit once the build is scheduled",
+					Destination: &async,
+				},
+				cli.StringFlag{
+					Name:        "transport",
+					Usage:       "Specify the transport method to transfer the build results",
+					Destination: &transport,
+					Value:       types.Scp,
+				},
+				cli.StringFlag{
+					Name:        "transport-user",
+					Usage:       "user to fetch the artifacts with",
+					Destination: &transportUser,
+					Value:       currentUser.Username,
+				},
+				cli.StringFlag{
+					Name:        "target, t",
+					Usage:       "the local directory where the result will be saved",
```

Comment:
Let's be consistent with our glossary and use the term "artifacts" instead of "results".

Reply:
[nit] also for consistency, s/the/The